website/content/en/docs/upgrading/upgrade-guide.md

```diff
@@ -36,60 +36,29 @@ WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE T
 ### Upgrading to `1.0.0`+
 
 {{% alert title="Warning" color="warning" %}}
-Karpenter `1.0.0` introduces v1 APIs, including _significant_ changes to the API and upgrade procedures for the Karpenter controllers. **Do not** upgrade to `1.0.0`+ without referencing the [v1 Migration Upgrade Procedure]({{<ref "v1-migration#upgrade-procedure" >}}).
-
-This version adds [conversion webhooks](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion) to automatically pull the v1 API version of previously applied v1beta1 NodePools, EC2NodeClasses, and NodeClaims. Karpenter will stop serving the v1beta1 API version at v1.1.0 and will drop the conversion webhooks at that time. Migrate all stored manifests to v1 API versions on Karpenter v1.0+.
+Karpenter `1.0.0` introduces the `v1` APIs and uses [conversion webhooks](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion) to support existing `v1beta1` APIs.
+**Do not** upgrade to `1.0.0`+ without following the [`v1` Migration Guide]({{<ref "./v1-migration.md">}}).
 {{% /alert %}}
 
-Below is the full changelog for v1, copied from the [v1 Migration Upgrade Procedure]({{<ref "v1-migration#upgrade-procedure" >}}).
```

Comment:
Removing this to prevent further instances of drift between the two sources. I don't think there's a ton of value in having this duplicated in the first place. Alternatively, we could create a separate file and embed it in both docs.